config/karma.saucelabs.js

```diff
@@ -119,19 +114,50 @@ module.exports = function(config) {
 
     customLaunchers: sauceLabsBrowsers,
 
-    reporters: ['spec', 'saucelabs'],
+    reporters: ['spec', 'summary', 'saucelabs'],
 
     port: 9876,
 
-    retryLimit: 3,
+    retryLimit: 0,
 
     // concurrency: 10,
 
+    client: {
+      mocha: {
+        timeout: 20000,
+        retries: 3,
+        grep:
+          'Crawler Support|local timestamp|raises error event|handles failures',
+        invert: true
+      }
+    },
+
+    browserConsoleLogOptions: { terminal: false },
+
+    specReporter: {
+      maxLogLines: 5,
+      suppressErrorSummary: true,
+      suppressFailed: false,
+      suppressPassed: true,
+      suppressSkipped: true,
+      showSpecTiming: true,
+      failFast: false
+    },
+
+    summaryReporter: {
+      show: 'failed',
+      specLength: 80,
+      overviewColumn: false
+    },
+
     sauceLabs: {
       tunnelIdentifier: process.env.TRAVIS_JOB_NUMBER,
       username: process.env.SAUCE_USERNAME,
       accessKey: process.env.SAUCE_ACCESS_KEY,
-      startConnect: true
+      startConnect: true,
+      connectOptions: {
+        noSslBumpDomains: 'all'
```

Comment:
This looks interesting!  Worth a comment?  I'm also curious if 'all' is the best option or if this should just be scoped to *.firebaseio.com or something?

Reply:
Commented below.